ext_emconf.php

```diff
@@ -15,7 +15,7 @@
     'description' => 'Moves the styles from CSS sheets into inline CSS, in order to comply with uncooperative email clients. Can be used as a post-processing stdWrap in TypoScript.',
     'category' => 'plugin',
     'shy' => 0,
-    'version' => '0.3.0-dev',
+    'version' => '7.0.0',
```

Comment:
What??? How can you justify to bump the version from 0.x to 7.0 ??? 1.0.0 (`-dev`) would be OK but I absolutely don't see how you could jump to an arbitrary 7.0.0.

Reply:
Was related to TYPO3 Version 7, but if you want to bump that, I'd even would propose 8.0.0.
That scheme is used by other extensions like gridelements too.

Reply:
On the other hand we've no separated version for TYPO3 and 9, so I'm fine with a version-scheme that is independent from TYPO3 version.